packages-exp/app-check-exp/src/internal-api.ts

```diff
@@ -136,12 +137,17 @@ export async function getToken(
 export function addTokenListener(
```

Comment:
Can `getToken()` and `addTokenListener()` take an appcheck instance instead of an app? then `platformLoggerProvider` doesn't need to be passed explicitly.

It's unclear why we are passing `platformLoggerProvider` unless you dive into the details. Using an appcheck instance sidesteps the problem and makes the code a bit cleaner IMO.

Reply:
Should be fixed.